packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -61,6 +66,16 @@ export async function processOutboundDocumentQueryResps({
     const contentTypes = docRefs.map(getContentTypeOrUnknown);
     const contentTypeCounts = getDocumentReferenceContentTypeCounts(contentTypes);
 
+    let successCount = 0;
+    let failureCount = 0;
+    for (const result of response) {
+      if (isSuccessfulOutboundDocQueryResponse(result)) {
+        successCount++;
+      } else {
+        failureCount++;
+      }
+    }
```

Comment:
Nit: Could we set this is as a function that returns the counts so its cleaner?  It gets called below `getDocumentReferenceContentTypeCounts`